dxtbx/format/nexus.py

```diff
@@ -1406,7 +1406,7 @@ def __init__(self, obj, beam):
 
         # Construct the detector model
         pixel_size = (fast_pixel_direction_value, slow_pixel_direction_value)
-        image_size = tuple(map(int, nx_module["data_size"][0:2]))
+        image_size = tuple(map(int, nx_module["data_size"][-2:]))
```

Comment:
Since we are aiming for clarity here I wonder if it is worth renaming `image_size` to `image_size_slow_fast` or similar? Discussion topic, since reviewing.

Reply:
Oh snap, you found a big bug.  That should have been

`        image_size = tuple(reversed(map(int, nx_module["data_size"][-2:])))`

Since the data is stored slow to fast, [-2:] gives us the image size in slow, fast order.  But dxtbx needs it in fast, slow order.  So if we add the reversed call, we get now match the NeXus spec.  But for one of the Eiger nexus files, print header now shows:

  image_size: {4362,4148}

Which is backwards...

Reply:
I didn't catch this because my use case is multipanel.  And since the Eiger is backwards, this backwards code worked...  I think the only fix is to special case it...

Reply:
Chat w/ @graeme-winter and he agrees, no choice but to special case it...  Commit incoming.